src/zappy_ai_src/ai_handle_response.py

```diff
@@ -99,10 +99,14 @@ def __countItem(self, tileStr : str, searchedItem : str) -> int:
         """ Count the occurence of the Item in the Tile """
         return tileStr.count(searchedItem)
 
-    def GetTile(self, x : int, y : int = 0) -> Tile:
+    def GetTile(self, x : int, y : int) -> Tile:
         """" Get one Tile of the Map """
         return self.__map[x + (y * y)]
 
+    def GetTile(self, x) -> Tile:
+        """" Get one Tile of the Map """
+        return self.__map[x]
```

Comment:
this is still bad. please rewrite the function or delete it

Reply:
Don't be silly billy

Reply:
The architecture is defined, I've created two functions to have the same results, because it's seems more easy to use, even if the second one is more representative